lib/app.dart

```diff
@@ -23,7 +31,45 @@ class App {
     _showContent(_currentPage);
     _renderProgress();
     container.classes.toggle("hidden", false);
-    interactive.Interactive(interactiveContainer);
+
+    initFirebase();
+  }
+
+  void initFirebase() async {
+    await fb.init();
+    fb.firebaseAuth.onAuthStateChanged.listen(_fbAuthChanged);
+    loginButton.onClick.listen((_) => fb.googleLogin());
+    logoutButton.onClick.listen((_) => fb.signout());
+  }
+
+  void _fbAuthChanged(firebase.User user) {
+    if (user == null) {
+      print("User signed out");
```

Comment:
Debugging prints (here and a couple below)? Can you either remove them, or turn into logging statements if you think it's useful to keep them around?

Reply:
Sorry i keep doing this - fixed with logger